server/client_manager.py

```diff
@@ -58,6 +58,8 @@ def __init__(self, server, transport, user_id, ipid):
             self.voting_at = 0
             self.is_checked = False
             self.websocket = None
+            self.active_player = False
```

Comment:
why is this needed?

Reply:
As of right now, we have no way to tell whether a client is just clicking around on the server browser or actually in the game.

This is used in the pull request for the New Join log that lists client version/ip/hwid, but this feature is a pretty critical backend thing that I'm really surprised you guys don't have already.

(New clients shouldn't be getting made just because someone clicked on the server name in the server browser regardless, but that itself is out of scope.)